### DISCUSSION TITLE
How to extract Attributes?

### DISCUSSION THREAD

**Discussion by leviskim17** [NaturalIntelligence/fast-xml-parser#642] [Q&A]
Here is my test code.

import {XMLParser} from 'fast-xml-parser';

const xmlDataStr = `<root a="nice" enabled="true" int="32"><a>wow</a></root>`;

var options = {
ignoreAttributes : false,
parseAttributeValue : true,
};

const parser = new XMLParser();
const jsonObj = parser.parse(xmlDataStr, options);
console.log(JSON.stringify(jsonObj, null, 2));

But the return cannot extract attributes.

`{
  "root": {
    "a": "wow"
  }

}`

Somebody has any idea? Thank u!

**Comment by amitguptagwl** [comment]
If you want to try experimental changes for v5, you can use `onAttribute` override method. You can find the code [here](https://github.com/NaturalIntelligence/fast-xml-parser/tree/master/src/v5). If I write any tutorial, I'll let you know.

**Comment by amitguptagwl** [comment]
Please check [@nodable/flexible-xml-parser](https://github.com/nodable/flexible-xml-parser) once
